[BUG] Edge install failed (Cannot install msedge on win32)

@mxschmitt this appears to be happening again right now:
```
##[error] [20:31:44.115] Firefox 117.0 (playwright build v1424) downloaded to C:\Users\cloudtest\AppData\Local\ms-playwright\firefox-1424
##[error] [20:31:44.116] Downloading Webkit 17.0 (playwright build v1908) from https://playwright.azureedge.net/builds/webkit/1908/webkit-win64.zip
##[error] [20:31:44.366] |                                                                                |   0% of 45.9 Mb
##[error] [20:31:44.436] |■■■■■■■■                                                                        |  10% of 45.9 Mb
##[error] [20:31:44.479] |■■■■■■■■■■■■■■■■                                                                |  20% of 45.9 Mb
##[error] [20:31:44.522] |■■■■■■■■■■■■■■■■■■■■■■■■                                                        |  30% of 45.9 Mb
##[error] [20:31:44.555] |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                                |  40% of 45.9 Mb
##[error] [20:31:44.586] |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                        |  50% of 45.9 Mb
##[error] [20:31:44.627] |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                |  60% of 45.9 Mb
##[error] [20:31:44.687] |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                        |  70% of 45.9 Mb
##[error] [20:31:44.734] |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                |  80% of 45.9 Mb
##[error] [20:31:44.783] |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■        |  90% of 45.9 Mb
##[error] [20:31:44.856] |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■| 100% of 45.9 Mb
##[error] [20:31:49.479] Webkit 17.0 (playwright build v1908) downloaded to C:\Users\cloudtest\AppData\Local\ms-playwright\webkit-1908
##[error] [20:31:49.585] Failed to install browsers
##[error] [20:31:49.585] Error: Cannot install msedge on win32
```

--- a/packages/playwright-core/src/server/registry/index.ts
+++ b/packages/playwright-core/src/server/registry/index.ts
@@ -918,7 +918,7 @@ export class Registry {
         darwin: { platform: 'MacOS', arch: 'universal', artifact: 'pkg' },
         win32: { platform: 'Windows', arch: 'x64', artifact: 'msi' },
       } as any)[process.platform];
-      const release = searchConfig ? product.releases.find((release: any) => release.platform === searchConfig.platform && release.architecture === searchConfig.arch) : null;
+      const release = searchConfig ? product.releases.find((release: any) => release.platform === searchConfig.platform && release.architecture === searchConfig.arch && release.artifacts.length > 0) : null;
       const artifact = release ? release.artifacts.find((artifact: any) => artifact.artifactname === searchConfig.artifact) : null;
       if (artifact)
         scriptArgs.push(artifact.location /* url */);

